src/content/docs/en/guides/testing.mdx

```diff
@@ -231,20 +231,20 @@ Here is an example of the configuration and commands required when using Yarn:
 1. Add a test script to your `package.json` file in the project root, such as `"test:e2e": "playwright test"`.
 1. In `playwright.config.ts`, add the `webServer` object and update the command value to `yarn preview`.
 
-    ```js title="playwright.config.ts" ins={3-8} "yarn preview"
-    import type { PlaywrightTestConfig } from '@playwright/test';
-    const config: PlaywrightTestConfig = {
+    ```js title="playwright.config.ts" ins={4-9} "yarn preview"
+    import { defineConfig } from '@playwright/test';
+    
+    export default defineConfig({
       webServer: {
         command: 'yarn preview',
```

Comment:
@sarah11918, not entirely related to this PR, but do we 'recommend' `yarn` to run the commands here? Or should we just fall to PNPM or NPM which we use most in docs (imo)

Reply:
Great call out, Elian! This advanced user case was submitted by @chuckcarpenter as the setup they used. I know earlier in the page, for basic usage, we do include our normal package manager tabs for options. We do typically use NPM generically, all other things being equal. So I'd certainly be open to that change!

Reply:
@ElianCodes Would you be willing to make the suggested change here so I can just commit it and not have to go through the effort of manually adding you as a co-author to this PR?  :sweat_smile: